pkg/registry/cache_test.go

```diff
@@ -0,0 +1,35 @@
+package registry
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Could we use gomega? That's the preferred test library in this repo

Reply:
I thought we had a discussion that everyone hated gomega